index.html

```diff
@@ -52,6 +52,195 @@ <h4 class="social-title">LET'S CONNECT</h4>
+            </p>
+            <ul class="card-languages">
+              <li class="card-lang1">html</li>
+              <li class="card-lang2">css</li>
+              <li class="card-lang3">javascript</li>
+            </ul>
+            <form action="#" class="card-form">
+              <input
+                type="submit"
+                value="See Project"
+                class="card-submit-button"
+              />
+            </form>
+          </section>
+        </div>
+        <div class="card">
+          <header class="card">
+            <img
+              src="./images/snapshot4.png"
+              class="card-header-pic"
+              alt="Projects Header"
+            />
+          </header>
+          <section class="card-projects">
+            <h1 class="card-title-2">Multi-Post Stories</h1>
+            <ul class="card-job-desc">
+              <li class="card-company">CANOPY</li>
+              <li><img src="./images/Counter.png" alt="Counter" /></li>
+              <li class="card-job">Back End Dev</li>
+              <li><img src="./images/Counter.png" alt="Counter" /></li>
+              <li class="card-year">2015</li>
+            </ul>
+            <p class="card-persona">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="card-languages">
+              <li class="card-lang1">html</li>
+              <li class="card-lang2">css</li>
+              <li class="card-lang3">javascript</li>
+            </ul>
+            <form action="#" class="card-form">
+              <input
+                type="submit"
+                value="See Project"
+                class="card-submit-button"
+              />
+            </form>
+          </section>
+        </div>
+        <section class="about-myself">
+          <h1 class="about-myself-title">About Myself</h1>
+          <p class="about-me-resume">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hesitate to
+            contact me.
+          </p>
+          <h4 class="connect">LET'S CONNECT</h4>
+          <ul class="social-media">
+            <li>
+              <a href="#"><img src="./images/github.svg" alt="Github" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/linkedln.svg" alt="Linkedln" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/fingers.svg" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/twitter.svg" alt="Twitter" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/gmail.svg" alt="Gmail" /></a>
+            </li>
+          </ul>
+          <p class="get-resume">Get my resume</p>
```

Comment:
- [ ] It is required to wrap your `Get my resume` with `<button>` tag and should a link with `<a>` tag for semantic reason and it will give you code a professional look.

Reply:
You got it.